Modules/Core/Common/include/itkObjectFactoryBase.h

```diff
@@ -57,7 +57,7 @@ namespace itk
 
 // Forward reference because of private implementation
 class OverRideMap;
-struct ObjectFactoryBasePrivate;
+class ObjectFactoryBasePrivate;
```

Comment:
This commit is surprisingly small.

Reply:
Yes, tracing and debugging took much longer than writing 😬